### PR TITLE
[website_crm] Migrated to new api.

### PR DESCRIPTION
**Pull request by rgo-odoo**
Task: https://www.odoo.com/web#id=12026&view_type=form&model=project.task&action=327
Pad: https://pad.odoo.com/p/r.TkQhEhvIYXjGJx6w

### CODE REVIEW

File: addons/website_crm/controllers/main.py [hunk 1]
```diff
@@ -60,13 +62,13 @@ def dict_to_str(title, dictvar):
         post_description = []  # Info to add after the message
         values = {}
 
-        values['medium_id'] = request.registry['ir.model.data'].xmlid_to_res_id(request.cr, SUPERUSER_ID, 'crm.crm_medium_website')
-        values['team_id'] = request.registry['ir.model.data'].xmlid_to_res_id(request.cr, SUPERUSER_ID, 'website.salesteam_website_sales')
+        values['medium_id']  = request.env['ir.model.data'].sudo().xmlid_to_res_id('crm.crm_medium_website')
+        values['team_id'] = request.env['ir.model.data'].sudo().xmlid_to_res_id('website.salesteam_website_sales')
```
Comment (atp-odoo) [inline addons/website_crm/controllers/main.py]: env.ref

File: addons/auth_oauth/controllers/__init__.py [hunk 2]
```diff
@@ -1 +1,3 @@
+# -*- coding: utf-8 -*-
```
Comment (msh-odoo) [inline addons/auth_oauth/controllers/__init__.py]: Add licence related stuff in __init__ and __openerp__

File: addons/website_crm/__openerp__.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```
Comment (msh-odoo) [inline addons/website_crm/__openerp__.py]: Add licence related stuff in init and manifest file

Comment (ajo-odoo) [inline addons/website_crm/__openerp__.py]: Done

File: addons/website_crm/tests/test_website_crm.py [hunk 4]
```diff
@@ -1,3 +1,6 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
 from openerp.api import Environment
```
Comment (msh-odoo) [inline addons/website_crm/tests/test_website_crm.py]: from odoo, also change below test case

File: addons/website_crm/models/crm_lead.py [hunk 5]
```diff
@@ -1,4 +1,8 @@
-from openerp import models, SUPERUSER_ID
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, SUPERUSER_ID
```
Comment (hmo-odoo) [inline addons/website_crm/models/crm_lead.py]: use api.environment instead of request.registry.

**Comment by tde-banana-odoo** [comment]
Done in master at https://github.com/odoo/odoo/commit/3275a90ff0ee71292b32a6005d2b1a289a09154b . Thanks !

**Pull request closed without merging**